src/flow.c

```diff
@@ -378,6 +378,89 @@ static inline void FlowUpdateEthernet(ThreadVars *tv, DecodeThreadVars *dtv,
     }
 }
 
+#if defined(ENABLE_ETM)
+#include <math.h>
+#include <float.h>
+
+inline void FlowEncryptedTrafficFinalize(const Flow *f)
```

Comment:
this function has a lot of `const`, but ultimately does change the structure which seems to be bit of an anti pattern